src/main.js

```diff
@@ -30,7 +31,11 @@ Vue.use(ImpressoLayout);
 if (process.env.VUE_APP_TAWK_TO_SITE_ID) {
   Vue.use(TawkTo, { siteId: process.env.VUE_APP_TAWK_TO_SITE_ID });
 }
-
+Vue.use(VueGtag, {
```

Comment:
We probably want to introduce an `if` condition here and not set the tag if the environment variable is empty.